src/Symfony/Bridge/Twig/Command/LintCommand.php

```diff
@@ -88,7 +88,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $io = new SymfonyStyle($input, $output);
 
         if (null === $twig = $this->getTwigEnvironment()) {
-            $io->error('The Twig environment needs to be set.');
+            $io->getErrorStyle()->error('The Twig environment needs to be set.');
```

Comment:
This will fail when the bridge is used with older versions (before 3.3) of the Console component which is completely valid.